azure_jumpstart_arcbox/artifacts/Bootstrap.ps1

```diff
@@ -121,6 +121,20 @@ if ($flavor -eq "DataOps") {
 }
 
 # Installing tools
+
+Write-Header "Installing PowerShell 7"
```

Comment:
Checking official docs, the recommended installation is to go with Winget: https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.4#install-powershell-using-winget-recommended -> Any reason why not to use winget? Seems to be simpler

Reply:
Good question. The reason is that WinGet must be bootstrapped first, and the easiest way to do that on Server SKUs is to use `Repair-WinGetPackageManager.` That command will install both the WinGet CLI, but the gotcha is that the command only works on PowerShell 7 - not when run in Windows PowerShell. Hence, we get a catch 22 situation. For all other packages, we plan to leverage WinGet (coming in separate PR).